convertCSV.js

```diff
@@ -1,34 +1,47 @@
 class CsvPhraseImporter {
 
   phrases = []
+
+  fileName = ""
   
   constructor(event) {
     this.readFile(event)
   }
   
   readFile(event) {
+
+    // Since there is only one file able to be inputted at a time, index 0 is the only array index
     let file = event.target.files[0]
+    this.fileName = event.target.files[0].name
     
     const reader = new FileReader();
     reader.addEventListener('load', (event) => {
-      //console.log(csv)
       this.csvToPhrase(event.target.result)
     });
     reader.readAsText(file);
   }
   
   csvToPhrase(result) {
     let rawParseResults = Papa.parse(result)
+
+    /*
+      Note: gets each *column* from each *row*
+    */
     rawParseResults.data.forEach((column) => {
       this.phrases.push(new Phrase(column[0], column[1], column[2]))
     })
   }
 }
 
-let phraseImporters = []
+let phraseImporters = [standardPhrases]
```

Comment:
Is is a list of phrase importers or just a list of phrase-collection?

Reply:
My intention was always to have it as a phrase collection.

Reply:
```suggestion
let phraseCollectionList = [standardPhrases]
```